release_notes/ocp-4-15-release-notes.adoc

```diff
@@ -1272,6 +1272,34 @@ With this release, instead of returning an overflow `servfail` error, the CoreDN
 [id="ocp-4-15-olm-bug-fixes"]
 ==== Operator Lifecycle Manager (OLM)
 
+* After installing an Operator, if the catalog becomes unavailable, the subscription for the Operator is updated with a `ResolutionFailed` status condition. However, after the catalog became available again, the `ResolutionFailed` status was not cleared. With this update, this status is now cleared from the subscription after the catalog becomes available, as expected. (link:https://issues.redhat.com/browse/OCPBUGS-29116[*OCPBUGS-29116*])
+
+* With this update, OLM performs a best-effort verification that existing custom resources (CRs) are not invalidated when you install an updated custom resource definition (CRD). (link:https://issues.redhat.com/browse/OCPBUGS-18948[*OCPBUGS-18948*])
+
+* Before this update, an Operator's install plan displayed duplicate values in the `clusterSeviceVersionNames` field. This update removes the duplicate values. (link:https://issues.redhat.com/browse/OCPBUGS-17408[*OCPBUGS-17408*])
+
+* Before this update, if you created an Operator group with same name as a previously existing cluster role, Operator Lifecycle Manager (OLM) would overwrite the cluster roles. With this fix, OLM generates a unique cluster role name for every Operator group by using the following syntax:
++
+.Naming syntax
++
+[source,text]
+----
+olm.og.<operator_group_name>.<admin_edit_or_view>-<hash_value>
+----
++
+For more information, see xref:../operators/understanding/olm/olm-understanding-operatorgroups.adoc#olm-operatorgroups-rbac_olm-understanding-operatorgroups[Operator groups]. (link:https://issues.redhat.com/browse/OCPBUGS-14698[*OCPBUGS-14698*])
+
+* Previously, if an Operator installation or upgrade took longer than 10 minutes, the operation could fail with the following error:
++
+[source,text]
+====
+Bundle unpacking failed. Reason: DeadlineExceeded, and Message: Job was active longer than specified deadline".
+====
++
+This issue occurred because Operator Lifecycle Manager (OLM) had a bundle unpacking job that was configured with a timeout of 600 seconds. Bundle unpack jobs could fail due to network or configuration issues in the cluster that might be transient or resolved with user intervention. With this bug fix, OLM automates the recreation of failed unpack jobs indefinitely by default.
```

Comment:
See https://www.ibm.com/docs/en/ibm-style?topic=word-usage#r for "re-create". I am not aware of formal guidance for "due to" but I think "because of" is better